net/nft-qos/files/nft-qos.config

```diff
@@ -26,7 +26,7 @@
 
 config default default
 	# Enable Flag for limit rate
-	option limit_enable '1'
+	option limit_enable '0'
```

Comment:
Why do you change the default config her?

Reply:
I think this option should be enabled by user only. -- anyway I reverted it back to 1 in the new commit. Thanks.